hnn_core/network.py

```diff
@@ -412,6 +427,15 @@ def add_poisson_drive(self, name, *, tstart=0, tstop=None, rate_constant,
             cell type with non-zero AMPA or NMDA weights.
         location : str
             Target location of synapses ('distal' or 'proximal')
+        n_drive_cells : int | 'n_cells'
```

Comment:
I wonder if the alternate argument `'cell_specific'` might provide the least confusion? Because you could  pass an int that is equal to `n_cells`, but as it's described in the docstring (which is really well written!) the connectivity pattern is really the distinguishing factor.

Reply:
See what you think after https://github.com/jonescompneurolab/hnn-core/pull/383/commits/1dd1fc87c43d31520fa7701974a2f8f4b54e036b, which makes the `n_drive_cells` argument consistent across `add_xxx_drive()` methods and hopefully clarifies that that 1-to-1 connectivity pattern is only implemented  when `n_drive_cells=='n_cells'`. I've gone back and forth as to whether `n_cells` is the right term here. Let me know what you think.